### ISSUE TITLE
ctags-lsp breaks when in insert mode.

### ISSUE THREAD

**Issue by ZohnyZeyad** [netmute/ctags-lsp.nvim#1]
I've setup ctags-lsp with neovim (no distros) using lspconfig (I also use blink.cmp) and whenever I try to type something with insert mode, I get this error:

![image](https://github.com/user-attachments/assets/21df8024-0f54-4706-94af-0abb25c128f1)

At first I thought it's a problem with the language/project I have open so I setup the lsp to work only with Lua files and I tried it on my own neovim config, the issue still persists.

Neovim version:

> NVIM v0.10.2
> Build type: Release
> LuaJIT 2.1.1713484068

ctags version:

> Universal Ctags 6.1.0, Copyright (C) 2015-2023 Universal Ctags Team
> Universal Ctags is derived from Exuberant Ctags.
> Exuberant Ctags 5.8, Copyright (C) 1996-2009 Darren Hiebert
>   Compiled: Dec 14 2024, 15:21:58
>   URL: https://ctags.io/
>   Output version: 0.0
>   Optional compiled features: +wildcards, +regex, +iconv, +option-directory, +xpath, +json, +interactive, +yaml, +packcc, +optscript, +pcre2

plugin version:

`$ brew info netmute/tap/ctags-lsp`
> ==> netmute/tap/ctags-lsp: stable 0.6.0
> LSP implementation using universal-ctags as backend
> https://github.com/netmute/ctags-lsp
> Installed
> /home/linuxbrew/.linuxbrew/Cellar/ctags-lsp/0.6.0 (6 files, 2MB) *
>   Built from source on 2024-12-21 at 21:04:21
> From: https://github.com/netmute/homebrew-tap/blob/HEAD/Formula/ctags-lsp.rb
> ==> Dependencies
> Required: universal-ctags ✔

**Comment by ZohnyZeyad** [comment]
Additional info that may be useful:

```
-- lazy.nvim
{
    "neovim/nvim-lspconfig",
    dependencies = "netmute/ctags-lsp.nvim",
    config = function()
        require("lspconfig").ctags_lsp.setup({
           filetypes = { "lua" },
        })
    end,
}
```

**Comment by netmute** [comment]
Fixed in ctags-lsp v0.6.1
Happy Holidays!
